applications/us2-cell-to-spatial/public/index.html

```diff
@@ -17,42 +17,71 @@
 
-            <span id="upload-file">
-              <label class="button" for="file-input">Upload CSV</label>
-              <input id="file-input" style="display: none;" type="file" accept=".csv, .tsv"> 
-            </span>
+              <span id="upload-file">
+                <label class="btn dark-btn" for="file-input">Upload CSV</label>
+                <input id="file-input" style="display: none;" type="file" accept=".csv, .tsv">
+              </span>
 
-            <button id="remove-file" style="display: none;">Remove CSV</button>
-          </p>
-        </div>
-        <div class="card">
-          <h2>2. Select an organ</h2>
-          <p>Select the organ (kidney, skin, heart, etc.) for the uploaded cell summary.</p>
-          <p>
-            <select id="organ-input">
-              <option value="http://purl.obolibrary.org/obo/UBERON_0000948">heart</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0002113">kidney</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0000059">large intestine</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0002048">lung</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0001911">mammary gland</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0002097">skin</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0002108">small intestine</option>
-              <option value="http://purl.obolibrary.org/obo/UBERON_0000056">ureter</option>
-            </select>
-          </p>
-          <p>
-            <button id="submit-file" style="display: none;">Generate Predictions</button>
-          </p>
+              <button id="remove-file" style="display: none;" class="btn light-btn">Remove CSV</button>
+            </p>
+          </div>
+          <div class="card">
+            <h3>2. Select an organ</h3>
+            <p>Select the organ (kidney, skin, heart, etc.) for the uploaded cell summary.</p>
+              <!-- <select id="organ-input">
+                <option value="http://purl.obolibrary.org/obo/UBERON_0000948">heart</option>
+                <option value="">kidney</option>
+                <option value="http://purl.obolibrary.org/obo/UBERON_0000059">large intestine</option>
+                <option value="http://purl.obolibrary.org/obo/UBERON_0002048">lung</option>
+                <option value="http://purl.obolibrary.org/obo/UBERON_0001911">mammary gland</option>
+                <option value="http://purl.obolibrary.org/obo/UBERON_0002097">skin</option>
+                <option value="http://purl.obolibrary.org/obo/UBERON_0002108">small intestine</option>
+                <option value="http://purl.obolibrary.org/obo/UBERON_0000056">ureter</option>
+              </select> -->
+              <md-filled-select>
+                <md-select-option selected value="http://purl.obolibrary.org/obo/UBERON_0000948">
```

Comment:
This will need to be generated in the code like before. Did the approach used for creating the select not work here? it should.

Reply:
Created a new function in javascript file that gets supported organs and constructs options accordingly. (styling work in progress)